README.md

```diff
@@ -1,104 +1,210 @@
+## QHub Cloud
+The cloud version of QHub is built using [Terraform](https://www.terraform.io/), [Helm](https://helm.sh/), and 
+[GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions).
 
-1. initialize
-2. render
-3. deploy
+### Cloud Providers
+QHub offers out-of-the-box support for [Digital Ocean], Amazon [AWS] and [GCP] (Google Cloud Provider). Although by 
+default, as long as the chosen provider offers Kubernetes services and has a Terraform module, it is possible to 
+configure QHub to deploy your project.
 
-these steps realized using the `qhub` CLI.
 
-### initialize configurations
+![image](docs/images/brand-diagram.png "architecture diagram")
 
-        qhub init do
-        qhub init aws
-        qhub init gcp
+For more details, check out the release [blog post](https://www.quansight.com/post/announcing-qhub).
 
-The `qhub init` command will generate configuration files for that service. The configutation files can be tailored to the needs of your organization. Each file specifies general project information, security, infrastructure settings, computational resource profiles and data science environments. See documentation on modifying your configuration file for all of the cloud providers: [Configuration File](https://github.com/Quansight/qhub/blob/master/docs/docs/aws/configuration.md) 
+## Usage
+### Installation pre-requisites
+* QHub is supported by macOS and Linux operating systems (Windows is **NOT** currently supported);
+* We recommend the adoption of virtual environments (`conda`, `pipenv` or `venv`) for successful usage. 
+
+### Install
+QHub's installation can be performed by using:
+* `conda`:
+  ```bash
+  conda install -c conda-forge qhub
+  ```
+  
+* or `pip` (instead):
+    ```bash
+    pip install qhub
+    ```  
+Once finished, you can check QHub's version (and additional CLI args) by typing:
+```bash
+qhub --help
+```
+If successful, the CLI output will be similar to the following:
 
-The configuration file is your user interface into deploying and scaling your data science environment. Each change triggers [Github Action] that will seamlessly update your infrastructure.
+![help_cli_output.png](docs/images/help_cli_output.png)
 
-![](docs/images/brand-diagram.png "architecture diagram")
+### Set up initialization
+#### 1. Cloud Provider
+The first required step is to **choose a Cloud Provider to host the project deployment**. The cloud installation is based
+on Kubernetes, but knowledge of Kubernetes is **NOT** required. By default, QHub uses DigitalOcean, AWS and GCP. If you 
+wish to use a different provider, make sure they support Kubernetes and Terraform and follow the suitable documentation
+to configure it.
 
-Check out the [`qhub` documentation][docs] for more detailed information.
+#### 2. Authentication (using OAuth and GitHub)
+In order to use GitHub actions, QHub will request a `client_id` and `client_secret`. To create those, you will need to 
```

Comment:
There's a ton of API keys involved in QHub, and I think they've been mixed up in this paragraph.

The Github[ "Personal Access Token"](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) is what github actions uses to automatically create commits when the `qhub-config.yaml` file is rendered, as well as kick of the github action to deploy. This is a single string that looks something like `qwioeurhgzkljdfvbnpsuer42903t213jkgbzdf9gwq834`

The Github [`client_id` and `client_secret`](https://github.com/settings/applications/new) can be used for [Oauth](https://en.wikipedia.org/wiki/Initiative_for_Open_Authentication). It will be a pair of a shorter string for the id, and a longer id for the secret.

[Auth0](auth0.com) can be used instead of the github authentication. The beauty of this option is that it has been automated by using `--oauth-provider auth0` [when the environment variables for auth0 have been set](https://github.com/Quansight/qhub/blob/master/docs/docs/step-by-step-walkthrough.md#22-auth0)

The person deploying has to decide to use github authentication or Auth0 authentication to verify users. They can't use both.

Reply:
Yup! Will fix this accordingly, thanks.

Reply:
This section was removed from the readme me :)